.github/CODEOWNERS

```diff
@@ -106,5 +106,16 @@ clang/test/AST/Interp/ @tbaederr
 # MLIR Sparsifier.
 /mlir/**/*SparseTensor*/ @aartbik @PeimingLiu @yinying-lisa-li @matthias-springer
 
+# MLIR NVGPU Dialect
+/mlir/**/NVGPU/ @grypp
+/mlir/**/NVGPUTo*/ @grypp
+/mlir/**/NVGPUTo*/ @grypp
+/mlir/**/sm90*/ @grypp
+
+# MLIR NVVM Dialect in MLIR
+/mlir/**/LLVMIR/BasicPtxBuilderInterface* @grypp
+/mlir/**/LLVMIR/NVVM* @grypp
+/mlir/**/NVVMToLLVM*/ @grypp
```

Comment:
That won't catch `mlir/include/mlir/Target/LLVM/NVVM`
Why not just do `mlir/**/NVVM*/`?